app/src/main/java/com/infomaniak/drive/utils/Extensions.kt

```diff
@@ -217,17 +217,18 @@ fun MaterialAutoCompleteTextView.setupAvailableShareableItems(
     itemList: List<Shareable>,
     notShareableIds: ArrayList<Int> = arrayListOf(),
     notShareableEmails: ArrayList<String> = arrayListOf(),
-    onDataPassed: (item: Shareable) -> Unit
+    onDataPassed: (item: Shareable) -> Unit,
 ): AvailableShareableItemsAdapter {
     setDropDownBackgroundResource(R.drawable.background_popup)
     val availableUsersAdapter = AvailableShareableItemsAdapter(
         context = context,
         itemList = ArrayList(itemList),
         notShareableIds = notShareableIds,
-        notShareableEmails = notShareableEmails
-    ) { item ->
-        onDataPassed(item)
-    }
+        getCurrentText = { text },
+        notShareableEmails = notShareableEmails,
```

Comment:
You could reorder these lines to keep the same ordering as inside the function declaration to make things easier to follow